deployment/Rules-Publish-TST-To-PRD.md

```diff
@@ -0,0 +1,48 @@
+# Rules Publish from TST to PRD
+
+1.	Remote desktop into **TDCI01SQL**
+
+2.	Using SQL Server Management Studio connect to the **TDCI01SQL** database server
+
+3.	Execute (*"Start Job at Step..."*) the SQL Server Agent Job *“Backup CodeDev”*
+
+![Backup CodeDev](../images/BackupCodeDevJob.png)
+
+4.	Open a command prompt and copy the file below into the window and hit enter
+    1.	C:\Infor\CopyDatabase - codeDEV to PRD.bat
+    ![Windows Batch File Copy Database](../images/WindowsBatchFileCopyDatabaseCodeDevToPrd.png)
+    *Note: the user running the bat file will need access to both the source and target folders referenced in the bat file*
+
+    2.	Verify the database has been copied to this shared drive *(\\\TDCI01SQLPROD\Backup)* and the Date Modified has changed to the current time.
+    ![TDCI01SQLPROD Backup File](../images/TDCI01SQLPROD-BackupFile.png)
+
+5. Execute (*"Start Job at Step..."*) the SQL Server Agent Job *“PCM Model DB Restore.CodeDEV to TestModel”*
+
+![Restore CodeDev To TestModel](../images/RestoreCodeDevToTestModel.png)
+
+6.	Using SQL Server Management Studio connect to the **TDCI01SQLPROD** database server
```

Comment:
I would make use of headings to say something like:

## Deploy to Test Rules Server

... numbered items

## Deploy to Production Rules Server

... numbered items